<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgOptionsJPEG</class>
 <widget class="QWidget" name="WdgOptionsJPEG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>390</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item column="0" row="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item column="0" row="1">
        <widget class="QCheckBox" name="progressive">
         <property name="text">
          <string comment="A progressive jpeg file can be displayed while loading.">Progressive</string>
         </property>
        </widget>
       </item>
       <item column="0" row="2">
        <widget class="QCheckBox" name="chkForceSRGB">
         <property name="text">
          <string>Force convert to sRGB</string>
         </property>
        </widget>
       </item>
       <item column="0" row="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item column="1" row="0">
          <widget class="KisDoubleSliderSpinBox" name="qualityLevel" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;These settings determine how much information is lost during compression. Low: small files, bad quality. High: big files, good quality.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item column="0" row="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Quality</string>
           </property>
          </widget>
         </item>
         <item column="0" row="1">
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Transparent pixel fill color:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item column="1" row="1">
          <widget class="KisColorButton" name="bnTransparencyFillColor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Background color to replace transparent pixels with.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item column="0" row="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item column="0" row="3">
        <widget class="QCheckBox" name="chkSaveProfile">
         <property name="text">
          <string>Save ICC Profile</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced quality</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item column="1" row="0">
          <widget class="KisDoubleSliderSpinBox" name="smoothLevel" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item column="0" row="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string comment="the result will be artificially smoothed to hide jpeg artifacts">Smooth:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item column="0" row="1">
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Subsampling:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item column="1" row="1">
          <widget class="QComboBox" name="subsampling">
           <item>
            <property name="text">
             <string>2x2, 1x1, 1x1 (smallest file)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2x1, 1x1, 1x1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1x2, 1x1, 1x1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1x1, 1x1, 1x1 (best quality)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="baseLineJPEG">
         <property name="text">
          <string>Force baseline JPEG</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="optimize">
         <property name="text">
          <string comment="Label of Optimize in JPEG Exporting dialogue">Optimize</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
      <layout class="QGridLayout">
       <item column="0" row="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Formats:</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="exif">
            <property name="text">
             <string>Exif</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="iptc">
            <property name="text">
             <string>IPTC</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xmp">
            <property name="text">
             <string>XMP</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item colspan="2" column="0" row="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>505</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item colspan="2" column="0" row="1">
        <widget class="QCheckBox" name="chkMetaData">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Store document metadata that is in the document information. This will override any layer metadata.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Store Document Metadata</string>
         </property>
        </widget>
       </item>
       <item column="1" row="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string comment="JPEG metadata filter">Filters:</string>
         </property>
         <layout class="QGridLayout">
          <item column="0" row="0">
           <widget class="QListView" name="metaDataFilters"/>
          </item>
         </layout>
        </widget>
       </item>
       <item colspan="2" column="0" row="3">
        <widget class="QCheckBox" name="chkAuthor">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add the author nickname and the first contact of the author profile. This is overridden by the anonymizer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Sign with Author Profile Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header>kis_color_button.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>progressive</tabstop>
  <tabstop>baseLineJPEG</tabstop>
  <tabstop>metaDataFilters</tabstop>
  <tabstop>exif</tabstop>
  <tabstop>iptc</tabstop>
  <tabstop>xmp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>